app/src/main/res/xml/video_audio_settings.xml

```diff
@@ -33,6 +33,13 @@
         android:summary="@string/show_higher_resolutions_summary"
         android:title="@string/show_higher_resolutions_title"/>
 
+    <SwitchPreference
+        android:defaultValue="true"
+        android:key="show_warning_live_stream_on_mobile"
+        android:summary="Show warning when playing live stream in background on mobile data"
```

Comment:
Please put the summary and title into the strings.xml file

Reply:
Done, thank you